metrics/metric.py

```
@@ -0,0 +1,887 @@
+                    # logger.info(f'Fetching {l} data from SemanticScholar.')
+                    return citation_count, influentionCC
+                    # raise ConnectionError
+
+                for item in r.json()['data']:
+                    if int(item['citationCount']) >= 0:
+                        citation_count.append(int(item['citationCount']))
+                        influentionCC.append(int(item['influentialCitationCount']))
+                        l += 1
+                    else:
+                        print(item['citationCount'])
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)